packages/s/surfer/package.yml

```diff
@@ -0,0 +1,25 @@
+name       : surfer
+version    : 0.3.0
+release    : 1
+source     :
+    - git|https://gitlab.com/surfer-project/surfer.git : v0.3.0
+homepage   : https://surfer-project.org/
+license    : EUPL-1.2
+component  : office.scientific
+summary    : An Extensible and Snappy Waveform Viewer
+description: |
+    A waveform viewer with a focus on a snappy usable interface, and extensibility.
+networking : yes
+builddeps  :
+    - pkgconfig(libssl)
+    - rust
+setup      : |
+    %cargo_fetch
+build      : |
+    %cargo_build
+install    : |
+    %cargo_install
+
+    # Install desktop file and appstream metainfo
+    install -Dm00644 %workdir%/surfer/assets/Surfer.desktop $installdir/usr/share/applications/com.gitlab.surferproject.surfer.desktop
+    install -Dm00644 $pkgfiles/com.gitlab.surferproject.surfer.metainfo.xml $installdir/sur/share/metainfo
```

Comment:
The metainfo install line has a couple of problems:

1. It's only creating the directory
2. It's creating `/sur`, not `/usr`